invenio_base/urls/helpers.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025 CERN.
+# Copyright (C) 2025 Northwestern University.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Urls helpers.
+
+Really just there to put invenio_url_for definition in a way that circumvents
+circular import.
+"""
+
+from flask import current_app
+
+
+def invenio_url_for(
+    endpoint,
+    *,
+    # _anchor = None,  # TODO
```

Comment:
I haven't see us use `_anchor` in urls, and I was just keeping it minimal stupid simple. We can eventually uncomment.